### PR TITLE
138 contact at the bottom of the footer

### PR DESCRIPTION
**Pull request by cparjaszewski**
Added contact field
Updated the echo-html component
Cleaned-up styles
Removed unnecessary firebase-document queries (in favor of using content)
echo-html component change applied to the overlay

### CODE REVIEW

File: app/elements/mdl-footer.html [hunk 1]
```diff
@@ -1,201 +1,208 @@
...
+        :host {
+          position: absolute;
+          left: 0;
+          width: 100%;
+          bottom: 0;
+        }
+      }
+
+      img {
+        list-style: none;
```
Comment (sojek) [inline app/elements/mdl-footer.html]: Why you set list-style for image selector?

File: app/elements/mdl-footer.html [hunk 2]
```diff
@@ -1,201 +1,208 @@
...
+      .footer_area[wide-layout] .one_third {
+        max-width: 400px;
+      }
+
+      .mdl-mini-footer {
+        background-color: white;
+      }
+
+      .footer_area[wide-layout] .mdl-mini-footer--left-section {
+        -webkit-order: 0;
```
Comment (sojek) [inline app/elements/mdl-footer.html]: You don't need prefixes because you have autoprefixer in gulp

File: app/elements/mdl-footer.html [hunk 3]
```diff
@@ -1,201 +1,208 @@
...
+        display: block;
+      }
+
+      .footer_area[wide-layout] .sponsor__link {
+        text-decoration: none;
+        display: inline-block;
+        text-align: left;
+      }
+
+      .footer_area[wide-layout] .sponsol__link {}
```
Comment (sojek) [inline app/elements/mdl-footer.html]: You have the same selector above

**Review by mindpal-teams — CHANGES_REQUESTED** [review]

File: app/elements/mdl-footer.html (hunk 1 above)
Comment (krzysztofp) [inline app/elements/mdl-footer.html]: Oh, I've just re-indented styles in this file, looks like some old style - probably I can delete this one.

File: app/elements/mdl-footer.html (hunk 3 above)
Comment (krzysztofp) [inline app/elements/mdl-footer.html]: I was just re-indenting styles in this file, but let me work on a quick fix for this.

**Review by mindpal-teams — APPROVED** [review]

**Pull request merged**